runtime/Cpp/README.md

```diff
@@ -35,12 +35,12 @@ The C++ target has been the work of the following people:
 
 ### Build + Usage Notes
 
-The minimum C++ version to compile the ANTLR C++ runtime with is C++11. The supplied projects can built the runtime either as static or dynamic library, as both 32bit and 64bit arch. The macOS project contains a target for iOS and can also be built using cmake (instead of XCode).
+The minimum C++ version to compile the ANTLR C++ runtime with is C++17. The supplied projects can built the runtime either as static or dynamic library, as both 32bit and 64bit arch. The macOS project contains a target for iOS and can also be built using cmake (instead of XCode).
```

Comment:
Is this minimum truly 17 now? Did we make a choice to do this or is this a new requirement?

Reply:
@parrt The constructor of ANTLRInputStream declared in ANTLRInputStream.h, uses std::string_view which is available only post [C++17](https://en.cppreference.com/w/cpp/string/basic_string_view) so the minimum required CMAKE_CXX_STANDARD to build the runtime library for C++ targets would be 17.